metaflow/plugins/aip/aip.py

```diff
@@ -1544,43 +1536,6 @@ def _create_metaflow_step_op(
         ).set_display_name(node.name)
         return container_op
 
-    def _create_workflow_uid_op(
```

Comment:
How come we don't need this any more? I presume because with the `{{workflow.uid}}` template it somehow magically fills in the string?